core/backend/ios/core-test-runner/core-test-runner-xcuitest/CoreTestRunnerXCUITest.swift

```diff
@@ -0,0 +1,27 @@
+import XCTest
```

Comment:
Can you add the copyright header? We should probably update the git hook to look at *.swift files as well.